templates/components/products/carousel.html

```diff
@@ -11,8 +11,9 @@
         {{> components/carousel/content-announcement}}
     {{/if}}
 
-    <div>
-        <div class="js-swiper-btn-prev">Previous</div>
-        <div class="js-swiper-btn-next">Next</div>
+    <div class="c-carousel__navigation">
+        <div class="js-swiper-btn-prev c-carousel__btn-prev"></div>
+        <div class="js-swiper-pagination c-carousel__pagination"></div>
+        <div class="js-swiper-btn-next c-carousel__btn-next"></div>
```

Comment:
@mgbelegu I usually put those `.js-` prefixed as last selectors inside the `class` attribute. That's a preference, you can ignore if you disagree ;).